src/lib/Scenes/Search/SearchResultList.tsx

```diff
@@ -1,13 +1,13 @@
 import { FadeIn } from "lib/Components/FadeIn"
 import { Join, Spacer } from "palette"
 import React from "react"
-import { SearchResult } from "./SearchResult"
+import { AutosuggestSearchResult } from "./AutosuggestSearchResult"
 
 export const SearchResultList: React.FC<{ results: React.ReactElement[] }> = ({ results }) => {
   return (
-    <Join separator={<Spacer mb="15px" />}>
+    <Join separator={<Spacer mb="20px" />}>
```

Comment:
suggestion(non-blocking): Can we change that to:
```suggestion
    <Join separator={<Spacer mb={2} />}>
```